src/org/rust/lang/core/psi/impl/mixin/RustBlockImplMixin.kt

```diff
@@ -6,16 +6,18 @@ import org.rust.lang.core.psi.RustBlock
 import org.rust.lang.core.psi.RustDeclStmt
 import org.rust.lang.core.psi.RustPatIdent
 import org.rust.lang.core.psi.impl.RustCompositeElementImpl
+import org.rust.lang.core.psi.util.boundIdentifiers
 import org.rust.lang.core.resolve.scope.RustResolveScope
 
 abstract class RustBlockImplMixin(node: ASTNode) : RustCompositeElementImpl(node)
         , RustBlock
-        , RustResolveScope{
+        , RustResolveScope {
 
     override fun listDeclarations(before: PsiElement): List<RustPatIdent> = children
             .filterIsInstance<RustDeclStmt>()
```

Comment:
fixed in master https://github.com/alexeykudinkin/intellij-rust/commit/b37e47abb15e0feeb7f76e09f76e704f1cc34552